src/2025h1/async.md

```diff
@@ -0,0 +1,151 @@
+# Bring the Async Rust experience closer to parity with sync Rust
+
+| Metadata |                                                              |
+| -------- | ------------------------------------------------------------ |
+| Owner(s) | @tmandry     |
+| Teams    | [lang], [libs], [libs-api] |
```

Comment:
Types should be included, since most of these features require significant T-types buy-in.

Reply:
Good catch, done.